### ISSUE TITLE
Inventory Map: Hexagonal bins sum Site EUI for buildings -- is this useful?

### ISSUE THREAD

**Issue by RDmitchell** [SEED-platform/seed#2398]
### Expected Behavior
I would have thought that the color of the Hexagonal bins represented the **_average_** Site EUI for the buildings inside the bin
### Actual Behavior
The color of the Hexagonal bins represents the **_sum_** of the Site EUIs for all the buildings inside the bin.

I'm not sure that is a useful metric for users ? I think average or mean or ?? would be more useful. 
### Steps to Reproduce
Display the map for a set of data that have Site EUI values
### Instance Information
instance: seeddemostaging
SHA: 62ee5c4a6

**Comment by adrian-lara** [comment]
The suggestion of making it an average sounds good to me. I do also wonder if we should get user feedback on this before making changes. @RDmitchell What do you think?

Another idea for future improvement, I could also imagine it being useful to give users control of the range - letting them change the default min of 20 and max of 750.

**Comment by RDmitchell** [comment]
Yes to all your questions, ie, it would be good to get user feedback before making changes, and it would be good for users to have control over the settings, possibly including whether they want to look at sum, average, mean, etc.
